experiments/AMIP/components/land/bucket_init.jl

```diff
@@ -159,6 +159,46 @@ function ClimaLSM.initialize_drivers(a::CoupledRadiativeFluxes{FT}, coords) wher
     return (;)
 end
 
+"""
+    temp_anomaly_zonally_asymmetric(coord)
+
+Introduce a zonally asymmetric temperature IC anomaly.
+Currently, the amplicture of the anomaly is 0 so no anomaly is applied.
+"""
+function temp_anomaly_zonally_asymmetric(coord)
+    FT = eltype(coord)
+    radlat = coord.lat / FT(180) * pi
+    anom_ampl = FT(0) # this is zero, no anomaly
+    lat_0 = FT(60) / FT(180) * pi
+    lon_0 = FT(-90) / FT(180) * pi
+    radlon = coord.long / FT(180) * pi
+    stdev = FT(5) / FT(180) * pi
+
+    anom = anom_ampl * exp(-((radlat - lat_0)^2 / 2stdev^2 + (radlon - lon_0)^2 / 2stdev^2))
+    return anom
+end
```

Comment:
Please, document these functions: where they come from, what they mean, and so on.

You also don't need `FT`s when your expression involves integers and `coord`. 

Also, it seems that you are converting from degrees to radians. There's a much more readable built-in function for that: `deg2rad`.

Finally, this expression is identically 0 all the time because `anom_ampl` is 0. I'd recommend understanding what is this function trying to accomplish and checking that its behavior is the intended one.

Reply:
We can remove this case - it's too specific to idealized experiments, which we aren't planning to do using this release.